web_app/tests/test_user.py

```diff
@@ -0,0 +1,72 @@
+import httpx
+import pytest
+
+from web_app.api.serializers.transaction import UpdateUserContractRequest
+
+
+@pytest.fixture(scope="function")
+async def async_client():
+    async with httpx.AsyncClient(base_url="http://localhost:8000") as client:
+        yield client
+
+
+@pytest.mark.asyncio
+async def test_get_user_contract(async_client):
+    response = await async_client.get(
+        url="/api/get-user-contract",
+        params={
+            "wallet_id": "",
+        },
+    )
+    response_json = response.json()
+
+    assert response.status_code == 200
+    assert isinstance(response_json, int)
+
+
+@pytest.mark.asyncio
+async def test_check_user(async_client):
+    response = await async_client.get(
+        url="/api/check-user",
+        params={
+            "wallet_id": "",
```

Comment:
I think we should handle this case as error, because it can be empty `""`/`None`